src/foundation/Icon/Generated/IconArrowBidirectional.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { memo } from "react";
+import { GeneratedIconProps } from "@foundation/Icon/IconProps";
+import { IconSize } from "@foundation/Icon/IconSize";
+
+import IconArrowBidirectional12 from './IconArrowBidirectional12';;
```

Comment:
There seems to be an error in the template we use for generation which produces double semicolons.
```suggestion
import IconArrowBidirectional12 from './IconArrowBidirectional12';
```

Reply:
Fixed!